service-deployment/src/pylon/dockerfile

```diff
@@ -44,6 +44,9 @@ RUN wget http://www.zlib.net/zlib-1.2.11.tar.gz && \
 RUN wget https://www.openssl.org/source/openssl-1.1.0f.tar.gz && \
     tar -zxf openssl-1.1.0f.tar.gz
 
+# subs_filter
+RUN git clone git://github.com/yaoweibin/ngx_http_substitutions_filter_module.git
```

Comment:
>yaoweibin [](start = 31, length = 9)

Does subs_filter has an official git? Or this is the official git?

Reply:
This is the official git. The module is not a standard nginx module listed [here](http://nginx.org/en/docs/), but it is included in the list of officially supported modules from NGINX, Inc. and NGINX Plus Certified Modules from NGINX's partners (see here). In addition, the moudle has a BSD license (see its doc [here](https://github.com/yaoweibin/ngx_http_substitutions_filter_module/tree/master/doc#Copyright__License)) so we have sufficient freedom to control it.